### DISCUSSION TITLE
Where should I install Radix UI?

### DISCUSSION THREAD

**Discussion by ghost** [vercel/turborepo#5502] [Help]
Hello, this feels like a silly question but I'm struggling to determine the best place to install Radix UI.

I'm using the [Turborepo + Next.js with Tailwind example](https://github.com/vercel/turbo/tree/main/examples/with-tailwind) but wondering what the best practice is for adding Radix UI to the setup. Following radix's install guide, it is added as a dependency within the root package.json. Should it not live as a dependency within the packages/ui folder's package.json as a dependency?

First time playing with many of these tools and wondering if I am over thinking it.

Thanks for any insight you all can provide.

**Comment by prince-bansal** [comment]
Hi, did you find the solution?
